communications/meetings.md

```diff
@@ -7,58 +7,47 @@ Use the guides below as a starting point, and adapt for specific situations acco
 
-Guide:
-- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
-    - [How to use Zoom for meetings](https://docs.google.com/document/d/1WNqeFKXYv_ji_PUXBKq8nlD09ErqmB6VoWasPW-BUDI/edit?usp=sharing).
-- Create a meeting agenda on Hugo, provide sufficient time for participants to read through ahead of the meeting.
-- Create a meeting note on Hugo for discussion on meeting agenda, sync it to Slack to augment this experience.
-    - On Hugo or Slack, participants should write down briefing, information, knowledge, and ideas.
-    - Participants should also raise questions, problems, obstacles, unknowns, and any other types of issues. Try to provide a demonstratable context to them.
-    - Seek to resolve as many issues and directional conflicts as possible.
-- Edit meeting agenda to reflect the outcomes of the discussion meeting note.
-- Put a tentative timebox on each agenda item.
-
-Tips:
-- View "All" meetings on Hugo to fit the practice of having pre-meeting notes for discussions.
-
-![all meetings](https://user-images.githubusercontent.com/2837532/76875868-8580d100-6847-11ea-95c8-532b29ea6d6d.png)
-
-- "Import Agenda" can be used as a quick starting point for any meeting note, especially the one used for agenda discussions.
+Guide for Organizers:
+- Send a calendar invite to intended meeting attendees with Zoom/Hangouts Meet.
+    - [How to use Zoom for meetings?](https://docs.google.com/document/d/1WNqeFKXYv_ji_PUXBKq8nlD09ErqmB6VoWasPW-BUDI/edit?usp=sharing)
+- Prepare meeting notes on Hugo, provide as much context as possible. Sync it to a relevant Slack channel to prompt for pre-meeting discussions. ![notes](https://user-images.githubusercontent.com/2837532/103943174-9d3f8380-50ff-11eb-9377-b839d780e10a.png)
+    - Make sure links to all relevant external references (such as Google Docs, Zoom recordings, Zeplin designs, Notion notes, etc.) are comprehensively included in the Hugo notes and make sure that all the participants have access to those.
+- [Tip] Use Hugo's "Import Agenda" feature as a quick starting point for any meeting note, especially the one used for agenda discussions.
 
-<img width="290" alt="import agenda" src="https://user-images.githubusercontent.com/2837532/76816543-e58b5f00-67d6-11ea-893a-a411a4a24bf8.png">
-
-- Hugo syncs notes discussions as Slack threads (two-way). Participants can freely discuss on either platform.
-
-<img width="562" alt="sync" src="https://user-images.githubusercontent.com/2837532/76794286-0d5dd100-679d-11ea-80b9-2077e2c6ce3f.png">
+Guide for Participants:
+- Go through relevant context, examples (where applicable):
+    - Meeting notes and their associated pre-meeting discussions prepared by the organizer
+    - Summary of pre-meeting discussions through the use of the `/notes` command. ![notes-discussion-summary](https://user-images.githubusercontent.com/2837532/103942564-9bc18b80-50fe-11eb-9c3b-12dde574c43e.png)
```

Comment:
there's `/notes template` and `/notes <link for the hugo note shared>`...I'm gonna write the legion ones and maybe we could link it in here?

Reply:
Yea was going to link it to that usage documentation perhaps under `workspace-automation` section. Same as for the dev journal automation part.